### ISSUE TITLE
piculet should respect additional AWS cli environment variables

### ISSUE THREAD

**Issue by emiller42** [codenize-tools/piculet#30]
It would be nice if the tool would make use of additional environment variables as outlined [here](http://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html)

I'm specifically thinking of the following:

- `AWS_CONFIG_FILE`: equivalent to `--credentials-path`
- `AWS_DEFAULT_PROFILE`: equivalent to `--profile`
- `AWS_DEFAULT_REGION`: equivalent to `--region`

If a user does not specify the above in CLI arguments when invoking piculet, then it should fall-back to any of the above ENV variables if they exist.  

End result is that if I already have my desired profile configured in my shell session, I don't need to explicitly tell piculet what profile to use.  

# Examples

## Current state:
```
$ export AWS_DEFAULT_PROFILE=development
$ piculet -a --dry-run
Apply `Groupfile` to SecurityGroup (dry-run)
[ERROR]
Missing Credentials.
$ piculet -p development -a --dry-run
Apply `Groupfile` to SecurityGroup (dry-run)
No change
```

## Desired state:
```
$ export AWS_DEFAULT_PROFILE=development
$ piculet -a --dry-run
Apply `Groupfile` to SecurityGroup (dry-run)
No change
```

**Comment by k1LoW** [comment]
Fixed by #31

**Comment by winebarrel** [comment]
Thanks!
